### PR TITLE
Adds More Guns to Armory & Removes Erroneous Holster Flag for Warden's Shotgun

### PR DESCRIPTION
**Pull request by Captain277**
## About The Pull Request

1. **Adds Guns to Armory.**
2. **Removes Holster Flag from Warden's Shotgun.**

## Why It's Good For The Game

1. _Adds eight spare tasers and four more LTL gunboxes to the Armory._
3. _This is apparently my fault. It appears to have come over with the initial port, but it's possible I assigned the flag myself. Either way, it's not an intended feature and I'm surprised it took this long for anyone to mention it._

## Changelog

:cl:
add: Adds more guns to the Armory.
del: Removes Warden's Shotgun's holster flag.
/:cl:

### CODE REVIEW

**Comment by lunar-crater-ex** [comment]
...okay, why does the already incredibly full and underused armory need More?

File: code/modules/projectiles/guns/projectile/shotgun.dm [hunk 1]
```diff
@@ -74,7 +74,7 @@
 	name = "warden's shotgun"
 	desc = "A heavily modified Hephaestus Industries KS-40. This version bears multiple after-market mods, including a laser sight to help compensate for its shortened stock. 'Property of the Warden' has been etched into the side of the reciever. Uses 12g rounds."
 	icon_state = "shotgun_w"
-	slot_flags = SLOT_BELT|SLOT_HOLSTER|SLOT_BACK
```
Comment (Zandario) [inline code/modules/projectiles/guns/projectile/shotgun.dm:77]: Did you mean to do this? Cause you have this PR: https://github.com/Citadel-Station-13/Citadel-Station-13-RP/pull/4807/files

**Comment by Captain277** [comment]
> ...okay, why does the already incredibly full and underused armory need More?

Because this adds standard issue firearms. These guns were originally stocked in Security vendors which were readily available in the Security break room. I had an issue with that implementation, and always promised that the weapons would be moved to the Armory, rather than being put in an open vending machine.

It's been a few months due to conflicting map edits, but thanks to Map Merge I can finally return and fulfill that promise. The Armory is "incredibly full" of lethal weaponry for various situations, but it is understocked for day-to-day concerns like basic taser distribution and replacement.

File: _maps/map_files/rift/rift-05-surface2.dmm [hunk 2]
```diff
@@ -22244,6 +22276,16 @@
...
+/obj/item/gunbox/lethal{
+	pixel_y = -4
+	},
+/obj/item/gunbox/lethal{
+	pixel_y = 8
+	},
+/obj/item/gunbox/lethal{
+	pixel_y = 4
+	},
+/obj/item/gunbox/lethal,
```
Comment (Shadowtail117) [inline _maps/map_files/rift/rift-05-surface2.dmm:22288]: Your PR description says you want to add less-than-lethal boxes. If so, remove the `/lethal`:
```suggestion
/obj/item/gunbox{
	pixel_y = -4
	},
/obj/item/gunbox{
	pixel_y = 8
	},
/obj/item/gunbox{
	pixel_y = 4
	},
/obj/item/gunbox,
```

**Review by Shadowtail117 — CHANGES_REQUESTED** [review]
Also, if you want to move security sidearms from the vendor to the armory, you may as well remove the .45 pistols from the SecTech as well. Alternatively put them in the officers' lockers so they don't need a warden/HOS to get their rubbers for them.

**Review by silicons — APPROVED** [review]
im just merging this because im redoing armory in a few weeks anyways

**Pull request merged**